docs/charts.txt

```diff
@@ -0,0 +1,64 @@
+https://stackoverflow.com/questions/23145621/how-to-publish-pages-on-github
+
+# init gh-pages
+git checkout --orphan gh-pages
+git rm -rf .
+echo "seed-chart" > index.html
+git add index.html
+git commit -a -m "add index.html"
+git push origin -u gh-pages
+
+# reset gh-pages
+git push origin :gh-pages
+git branch -D gh-pages
+
+https://github.com/edgelevel/gitops-k8s/settings
+
+Section: GitHub Pages
+Source: "gh-pages branch"
+Tick: "Enforce HTTPS"
+
+https://edgelevel.github.io/gitops-k8s
+https://edgelevel.github.io/gitops-k8s/index.html
+
+---
+
+https://stackoverflow.com/questions/13897717/push-commits-to-another-branch
+https://stackoverflow.com/questions/23145621/how-to-publish-pages-on-github
+https://help.github.com/en/articles/creating-project-pages-using-the-command-line
+
+---
+
+TODO master branch
```

Comment:
I don't understand this TODO and can you explaining putting stuff in `.txt` files

Reply:
this are notes (`.txt` instead of `.md`) and the example below is using the `seed` branch, i was taking personal notes to understand how automate it with travis and keep track of what i need to document in the readme once is stable. this file is not linked anywhere else and I do not want to keep it locally only. I'm going to delete it once is done